docs/datasets_and_tables/gtfs_rt.md

```diff
@@ -46,7 +47,14 @@ broadly useful across the org should live in `views`.
 
 ### Extraction
 
-Extraction of GTFS RT feeds is handled by the [gtfs-rt-archive service](../services/gtfs-rt-archive.md).
+Extraction of GTFS RT feeds is handled by the [gtfs-rt-archive service](../services/gtfs-rt-archive.md).Within this service is some basic logging functionality that exports the logs to an external table 'std.out' that reads the data in this bucket.
```

Comment:
I don't believe this sufficiently explains how the the logs from the archiver service are automagically saved to Google Cloud and analyzeable within the logging dashboard. However, that seems out of scope for this PR.